### PR TITLE
Reverse String in Python

### PR DESCRIPTION
**Pull request by Jahnavi-Jonnalagadda**
**Fixes issue:** #4732 

**Changes:** Added file which reverses words in a string in Python

### CODE REVIEW

File: Reverse_word_string.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+// Reversing a string in Python
```
Comment (arnavb) [inline Reverse_word_string.py]: Please fix the comments in this file.

File: Reverse_word_string.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+CODE:
```
Comment (arnavb) [inline Reverse_word_string.py]:
```suggestion
```

File: Reverse_word_string.py (hunk 1 above)
Comment (arnavb) [inline Reverse_word_string.py]:
```suggestion
# Reversing a string in Python
```

File: Reverse_word_string.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    else:
+        l.append(s1)
+        s1=""
+print("The Reversed String: ", *l[::-1])
+
+// INPUT
+Enter a string: Code is Life
+
+// OUTPUT:
+The Reversed String: Life is Code
```
Comment (arnavb) [inline Reverse_word_string.py]:
```suggestion
# INPUT
# Enter a string: Code is Life
#
# OUTPUT:
# The Reversed String: Life is Code
```

File: Reverse_word_string.py [hunk 4]
```diff
@@ -0,0 +1,21 @@
+// Reversing a string in Python
+
+CODE:
+s = input("Enter a string: ")
+s1 = ""
+m = list(s)
+m.append(" ")
+l = []
+for i in range(len(m)):
```
Comment (arnavb) [inline Reverse_word_string.py]: Reformat this file with black.

File: Reverse_word_string.py [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+    if(m[i]!=" " ):
```
Comment (arnavb) [inline Reverse_word_string.py]: No need for parentheses.

File: Reverse_word_string.py [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+        s1 = s1 + m[i]
```
Comment (arnavb) [inline Reverse_word_string.py]:
```suggestion
        s1 += m[i]
```

File: Reverse_word_string.py (hunk 1 above)
Comment (arnavb) [inline Reverse_word_string.py]: Move this file to the correct directory as well.

File: Reverse_word_string.py (hunk 5 above)
Comment (Jahnavi-Jonnalagadda) [reply Reverse_word_string.py]: Made the changes.

File: Reverse_word_string.py (hunk 1 above)
Comment (Jahnavi-Jonnalagadda) [reply Reverse_word_string.py]: Could you please let me know in which directory should I place this?

Comment (arnavb) [reply Reverse_word_string.py]: @Jahnavi-Jonnalagadda move it to string algorithms: https://github.com/OpenGenus/cosmos/tree/master/code/string_algorithms/src.

Comment (arnavb) [reply Reverse_word_string.py]: Don't forget to reformat this file with black.

File: tree/master/code/string_algorithms/src/Reverse_word_string.py [hunk 7]
```diff
@@ -0,0 +1,20 @@
+# Reversing a string in Python
```
Comment (arnavb) [inline tree/master/code/string_algorithms/src/Reverse_word_string.py]: The file should be in its own dirrectory in its folder and both the folder/file name should be all lowercase, e.g `code/string_algorithms/src/reverse_word_string/reverse_word_string.py`

File: Reverse_word_string.py (hunk 1 above)
Comment (Jahnavi-Jonnalagadda) [reply Reverse_word_string.py]: How to reformat a file with black?

Comment (arnavb) [reply Reverse_word_string.py]: @Jahnavi-Jonnalagadda either install it with `pip install` or use it online [here](https://black.now.sh/).

Comment (Jahnavi-Jonnalagadda) [reply Reverse_word_string.py]: Unable to install black. Could you please guide me.

Comment (arnavb) [reply Reverse_word_string.py]: If you are unable to install it, use the online version. Just paste your code in the URL and put the modified code in this PR.

Comment (Jahnavi-Jonnalagadda) [reply Reverse_word_string.py]: Yes, made changes.

**Pull request merged**